db-auto-backup.py

```diff
@@ -128,7 +128,7 @@ def backup_redis(container: Container) -> str:
 
 BACKUP_PROVIDERS: list[BackupProvider] = [
     BackupProvider(
-        patterns=["postgres"], backup_method=backup_psql, file_extension="sql"
+        patterns=["postgres", "*pgvecto*", "*postgres*"], backup_method=backup_psql, file_extension="sql"
```

Comment:
Issue: I think these globs are a little too vague to be reliable. It'd be nicer to target the containers you're expecting more closely.